src/Plugin/resource/Field/ResourceFieldEntity.php

```diff
@@ -113,9 +115,11 @@ public function __construct(array $field, RequestInterface $request) {
     $this->wrapperMethodOnEntity = isset($field['wrapper_method_on_entity']) ? $field['wrapper_method_on_entity'] : $this->wrapperMethodOnEntity;
     $this->column = isset($field['column']) ? $field['column'] : $this->column;
     $this->imageStyles = isset($field['image_styles']) ? $field['image_styles'] : $this->imageStyles;
-    if (!empty($field['bundles'])) {
-      $this->bundles = $field['bundles'];
+    if (!empty($field['bundle'])) {
```

Comment:
This was not used until now, that's why we didn't realize that it made no sense to have multiple bundles in a field instance for a loaded entity.

Having a single bundle simplifies things greatly.